src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java

```diff
@@ -70,6 +70,12 @@ public enum State {
     )
     Withdrawn,
 
+    @CCD(
+        label = "Archived",
+        hint = "### Case number: ${hyphenatedCaseRef}\n ### ${applicant1LastName} and ${applicant2LastName}\n",
+        access = {DefaultStateAccess.class, SolicitorAccess.class}
```

Comment:
Should we use DefaultStateAccess ? It will then make the case available to CAA

Reply:
Sure I can remove this. Wasn't sure if it would cause caseworkers to be able to see it as it's a pre-submission state. Can't remember if caseworkers can only see post-sub cases.

If I remove it, will that effect the grantHistory tab for the event if I give caseworker history access?

Reply:
Discussed on slack. BA has said to remove all caseworker access for this event for now and if the business later confirm it's needed then a new ticket will spring up to fix this. Resolving this comment and the one below.